pkg/api/register.go

```diff
@@ -118,52 +118,67 @@ func AddToScheme(scheme *runtime.Scheme) {
 func (obj *Pod) GetObjectMeta() meta.Object                                  { return &obj.ObjectMeta }
 func (obj *Pod) GetObjectKind() unversioned.ObjectKind                       { return &obj.TypeMeta }
 func (obj *PodList) GetObjectKind() unversioned.ObjectKind                   { return &obj.TypeMeta }
+func (obj *PodList) GetListMeta() meta.List                                  { return &obj.ListMeta }
```

Comment:
Why do you need this function? Given what you're returning, you could just implement this on the ListMeta type and everyone could get it for free?